envpool/mujoco/dmc/mujoco_dmc_align_test.py

```diff
@@ -87,7 +89,7 @@ def run_align_check(
         for k in obs_spec:
           np.testing.assert_allclose(o0[k], getattr(o1, k)[0])
         np.testing.assert_allclose(ts0.step_type, ts1.step_type[0])
-        np.testing.assert_allclose(ts0.reward, ts1.reward[0])
+        np.testing.assert_allclose(ts0.reward, ts1.reward[0], rtol=1e-6)
```

Comment:
If I do not change the rtol from 1e-7 to 1e-6, the reward will fail. I am not sure if there is a bug in my code or if there exists a computation accuracy problem.